praw/__init__.py

```diff
@@ -379,6 +379,27 @@ def _request(self, url, params=None, data=None, files=None, auth=None,
         :returns: either the response body or the response object
 
         """
+        def build_key_items():
```

Comment:
Excellent! My only suggestion (which I know some of the other nested methods don't follow) is to explicitly pass in the values needed so the function can be moved elsewhere if necessary.